content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -92,8 +92,22 @@ the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
 `SupportPodPidsLimit` to work.
 {{< /note >}}
 
+## PID based eviction
+
+Eviction is the way to configure kubelet to start terminating a pod if it misbehaving.
+You can set eviction threshold via `pid.available` eviction signal
+(see [Configure Out of Resource Handling](content/en/docs/tasks/administer-cluster/out-of-resource.md)).
```

Comment:
- we may want to avoid using parenthesis when possible
- the link target should be `/docs/tasks/administer-cluster/out-of-resource/`
- it sounds like 'pid.available' is the only eviction signal ... although that is not true